src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/BarnetrygdInformasjonsbrevKanSøkeEøsMetadata.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import no.nav.familie.kontrakter.felles.Behandlingstema
+import no.nav.familie.kontrakter.felles.Fagsystem
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.dokarkiv.Dokumenttype
+import org.springframework.stereotype.Component
+
+@Component
+object BarnetrygdInformasjonsbrevKanSøkeEøsMetadata : Dokumentmetadata {
+
+    override val journalpostType: JournalpostType = JournalpostType.UTGAAENDE
+    override val fagsakSystem: Fagsystem = Fagsystem.BA
+    override val tema: Tema = Tema.BAR
+    override val behandlingstema: Behandlingstema? = null
```

Comment:
Burde behandlingstema være not null og settes til `Behandlingstema.Barnetilsyn`?

Reply:
Godt spørsmål. Ville tro `BarnetrygdEØS` er mer passende i såfall. Ser ut som det meste av Dokumentmetadata fra ba ikke har noe behandlingstema. Vet du hva som burde være her og hva det brukes til? Evt hvem man kan spørre?